com.vrcfury.vrcfury/SPS/sps_main.cginc

```diff
@@ -58,18 +58,34 @@ void sps_apply_real(inout float3 vertex, inout float3 normal, uint vertexId, ino
 			applyLerp = min(applyLerp, 1-entranceAngleTooSharp);
 		}
 		
+		const float hiltedSphereRadius = 0.3;
 		if (!isRing) {
 			// Uncancel if hilted in a hole
-			const float hiltedSphereRadius = 0.3;
 			const float inSphere = orfDistance > worldLength*hiltedSphereRadius ? 0 : 1;
 			//const float hilted = min(isBehind, inSphere);
 			//shrinkLerp = hilted;
 			const float hilted = inSphere;
 			applyLerp = max(applyLerp, hilted);
 		} else {
-			// Cancel if ring is near or behind base
-			const float isBehind = rootPos.z > 0 ? 0 : 1;
-			applyLerp = min(applyLerp, 1-isBehind);
+			// Align to axis if hilted in a ring
+			// Prevent orifice from getting too close to plug as that can introduce excessive roll
+			const float minDistance = worldLength * 0.2;
+			const float lerpDistance = minDistance;
+			
+			// Consider plane aligned to orifice
+			const float orfPerpDistance = dot(sps_normalize(-frontNormal), rootPos);
+			float shiftLerp = (orfPerpDistance > minDistance) | (orfPerpDistance < -worldLength * hiltedSphereRadius) ? 0 : 1;
```

Comment:
ambiguous order of operations. Add parens and use || rather than | (although ultimately it probably results in the same machine code)